doc(1.8.0, 1.7, 1.6): online volume expansion for encrypted volumes (#1026)
longhorn/website: content/docs/1.8.0/nodes-and-volumes/volumes/expansion.md

#### Encrypted volume

> Longhorn supports online expansion, and it depends on the Kubernetes:
> 
> - Kubernetes support [authenticated CSI storage resizing](https://kubernetes.io/blog/2023/12/15/csi-node-expand-secret-support-ga/) natively from v1.29.
> - From [Kubernetes v1.25 to v1.28](https://kubernetes.io/blog/2022/09/21/kubernetes-1-25-use-secrets-while-expanding-csi-volumes-on-node-alpha/), the feature gate `CSINodeExpandSecret` is required.
> 
> Online expansion for encrypted volume can be enabled by [specifying encryption parameters in the StorageClass](../../../advanced-resources/security/volume-encryption#setting-up-kubernetes-secrets-and-storageclasses):

Reviewer: ```suggestion
Longhorn support for online expansion depends on Kubernetes.

- Kubernetes natively supports [authenticated CSI storage resizing](https://kubernetes.io/blog/2023/12/15/csi-node-expand-secret-support-ga/) starting in v1.29.
- In [Kubernetes v1.25 to v1.28](https://kubernetes.io/blog/2022/09/21/kubernetes-1-25-use-secrets-while-expanding-csi-volumes-on-node-alpha/), the feature gate `CSINodeExpandSecret` is required.

You can enable online expansion for encrypted volumes by specifying the following [encryption parameters in the StorageClass](../../../advanced-resources/security/volume-encryption#setting-up-kubernetes-secrets-and-storageclasses):
```
